kite-data/kite-data-core/src/main/java/org/kitesdk/data/Datasets.java

```diff
@@ -225,14 +324,49 @@
   /**
    * Update a {@link Dataset} for the given dataset or view URI.
    * <p>
-   * URI formats are defined by {@code Dataset} implementations, but must begin
-   * with "dataset:" or "view:".
+   * You can add columns, remove columns, or change the data type of columns 
+   * in your dataset, provided you don't attempt a change that is incompatible
+   * with written data. Avro defines rules for compatible schema evolution. See 
+   * <a href="http://kitesdk.org/docs/current/Schema-Evolution.html">Schema
+   * Evolution</a>.
+   * <p>
+   * This method updates the dataset descriptor, so you can also add
+   * or change properties.
+   * <p>
+   * The recommended way to update a dataset descriptor is to build it
+   * based on an existing descriptor. Use
+   * {@link DatasetDescriptor.Builder(DatasetDescriptor descriptor)} to 
```

Comment:
Link tags use the type, but not the argument name. This should be `{@link DatasetDescriptor.Builder(DatasetDescriptor)}`